File: src/test/java/cernx/colt/matrix/tfloat/impl/SparseRCMFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class SparseRCMFloatMatrix2DViewTest extends SparseRCMFloatMatrix2DTest {

    public SparseRCMFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCMFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}

File: src/test/scala/cern/colt/matrix/tfloat/impl/SparseRCMFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.SparseRCMFloatMatrix2D

class SparseRCMFloatMatrix2DViewTest(arg0: String) extends SparseRCMFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseRCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseRCMFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseRCMFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
